summerofcode/2021.md

```diff
@@ -187,3 +187,11 @@ identity federation. Account for 1) multiple Kubernetes clusters using the same
 -	Recommended Skills: Go
 -	Mentor(s): Joe Kutner (@jkutner)
 -	Upstream Issue (URL): https://github.com/buildpacks/lifecycle/issues/555
+
+### in-toto
+#### Develop in-toto-rs (Rust) for integration with rebuilderd
+
+- Description: [rebuilderd](https://github.com/kpcyrd/rebuilderd) is a verification system for binary packages. It repeats the build process of a package in an identical environment and verifies that the package is identical. It is part of the [Reproducible Builds](https://reproducible-builds.org/) effort and can currently be used to [rebuild Arch Linux packages](https://tests.reproducible-builds.org/archlinux/archlinux.html). The rebuild must optionally generate in-toto link attestations which can be used to verify the entire process. To that end, the nascent [in-toto-rs](https://github.com/in-toto/in-toto-rs) library must be developed to enable this integration with rebuilderd.
+- Recommended Skills: Rust
+- Mentor(s): Santiago Torres-Arias (@SantiagoTorres)
+- Upstream Issue (URL): https://github.com/kpcyrd/rebuilderd/pull/22
```

Comment:
@SantiagoTorres @adityasaky  I’ve noticed this last-minute, but linked issues outside of the CNCF project’s GitHub are not allowed.

Please update this file with a new issue created within the in-toto GitHub organization.

Reply:
I'll open a separate PR to fix this shortly.

Reply:
Thanks!